homeassistant/components/mystrom/config_flow.py

```diff
@@ -0,0 +1,80 @@
+"""Config flow for myStrom integration."""
+from __future__ import annotations
+
+import ipaddress
+import logging
+import re
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_NAME
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import DEFAULT_NAME, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_NAME, default=DEFAULT_NAME): str,
```

Comment:
You shouldn't need to ask for a name. It should come from the device info, or use a default entry title.
```suggestion
```

Reply:
Removed